mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -144,6 +160,13 @@ void tearDown() {
             .forEach(queueName -> sender.delete(QueueSpecification.queue(queueName.asString())).block());
         sender.delete(ExchangeSpecification.exchange(MAILBOX_EVENT_EXCHANGE_NAME)).block();
         sender.close();
+
+        rabbitMQExtension.afterEach(null);
```

Comment:
Extract `null` in a constant named `NO_EXTENSION_CONTEXT`

Reply:
I will remove this commit and do what Rene proposed, seems like it is cleaner than this way.